### ISSUE TITLE
PHP Notice: Undefined offset: 0 on pull content page

### ISSUE THREAD

**Issue by grappler** [10up/distributor#402]
**Describe the bug**
`Notice: Undefined offset: 0 in /plugins/distributor/includes/pull-ui.php:394`

There is no check if there are ANY available post types.
```php
$connection_now->pull_post_types = \Distributor\Utils\available_pull_post_types( $connection_now, $connection_type );
$connection_now->pull_post_type = $connection_now->pull_post_types[0]['slug'];
```
https://github.com/10up/distributor/blob/1.4.1/includes/pull-ui.php#L394

**Steps to Reproduce**

1. Go to Pull Content page wp-admin/admin.php?page=pull
2. See PHP Notice

**Expected behavior**
No PHP Notice

**Comment by jeffpaul** [comment]
@grappler I'm milestoning this for our larger 2.0.0 release and will look to get it resolved then, thanks again for the feedback!

**Comment by jeffpaul** [comment]
@grappler I'm no longer seeing this PHP notice.  Are you still seeing it?  If so, can you confirm your WordPress, Distributor, and PHP versions?

**Comment by grappler** [comment]
@jeffpaul You can force the issue by including `add_filter( 'dt_available_pull_post_types', '__return_empty_array' );`

**Comment by dkotter** [comment]
@grappler Thanks for the additional information here. I've been able to reproduce this and have a fix in place in #703
